bindings/python/htcondor2/_submit.py

```diff
@@ -161,7 +163,8 @@ def __iter__(self):
 
 
     def __len__(self):
-        return len(self.keys())
+        keys = _submit_keys(self, self._handle)
+        return len(keys)
```

Comment:
Don't we want to return the number of keys here? This looks to be returning the number of characters in the key names (plus the NUL separators).

Reply:
Excellent catch.  Fixed.